code/parse/sexp/LuaAISEXP.cpp

```diff
@@ -0,0 +1,150 @@
+//
+//
+#include "LuaAISEXP.h"
+
+#include "localization/localize.h"
+#include "parse/parselo.h"
+#include "parse/sexp/LuaSEXP.h"
+
+using namespace luacpp;
+
+namespace sexp {
+static const SCP_unordered_set<int> allowed_oswpt_parameters{ OPF_SHIP, OPF_WING, OPF_SHIP_POINT, OPF_SHIP_WING, OPF_SHIP_WING_WHOLETEAM, OPF_SHIP_WING_SHIPONTEAM_POINT, OPF_SHIP_WING_POINT, OPF_SHIP_WING_POINT_OR_NONE };
+
+LuaAISEXP::LuaAISEXP(const SCP_string& name) : DynamicSEXP(name) {
+}
+
+void LuaAISEXP::initialize() {
};

```

Comment:
There are a lot of `^M` characters in this file but not any of the other files.  Were there newline errors when this file was added?